protocol/x/clob/ante/clob.go

```diff
@@ -53,17 +54,11 @@ func (cd ClobDecorator) AnteHandle(
 		return next(ctx, tx, simulate)
 	}
 
-	// Ensure that if this is a clob message then that there is only one.
-	// If it isn't a clob message then pass to the next AnteHandler.
-	isSingleClobMsgTx, err := IsSingleClobMsgTx(tx)
-	if err != nil {
+	// Check if the transaction is a valid clob tx
+	if _, err := IsValidClobMsgTx(tx); err != nil {
```

Comment:
Implicit contract here: if `false` then we go through for loop below (no-op) and jump to `next(ctx, tx, simulate)`. Let's make it explicit to improve readability and avoid future bug that breaks the contract

```
	if !isClobMsg {
		return next(ctx, tx, simulate)
	}
```